ui-v2/app/validations/acl.js

```diff
@@ -2,5 +2,4 @@ import { validatePresence, validateLength } from 'ember-changeset-validations/va
 export default {
   Name: [validatePresence(true), validateLength({ min: 1 })],
```

Comment:
So dos the KV not have validations? Or they just weren't run?

Reply:
Yeah it does, but they only started running once you'd focussed a field :( not sure if that's because of me or the plugin, or just the use case.